src/components/formik-textarea/FormikTextarea.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * @class FormikTextarea
+ */
+
+import * as React from 'react';
+import styled from 'styled-components';
+import { loadStyleVariables } from "../../scripts/loadStyleVariables";
+
+type Props = {
+  field: {
+    name: string;
+  };
+  form: {
+    ['errors']: string;
+    ['touched']: boolean;
+  };
+
+  ['serverErrors']: string;
+
+  validationMessage: string;
+  label: string;
+  placeholderText: string;
+  error: boolean,
+  focussed: boolean,
+  disabled: boolean,
+  required?: any,
+  rows?: number,
+  cols?: number,
+  classNames?: string[],
+};
+
+const styleVariables = loadStyleVariables();
+
+const InputBoxRight = styled.div<{ error?: boolean; disabled?: boolean, focussed?: boolean }>`
+  border: ${props => props.error ? `2px solid ${styleVariables.red}` : `2px solid ${styleVariables.grayLight}`};
+  color: ${props => props.error ? `${styleVariables.red}` : `${styleVariables.grayLight}`};
+  padding: 10px 10px 10px 10px;
+  border-radius: 2.5px;
+  background: ${styleVariables.colorWhite};
+  opacity: ${props => props.disabled ? '0.5' : '1'};
+  outline: none;
+  display: flex;
+  justify-content: space-between;
+  height: 200px;
```

Comment:
Worden textarea's nooit hoger dan 200px? Anders beter via een prop aanpasbaar maken en laten defaulten op 200px.
Heb je ook voorzien dat de tekst die er in komt misschien langer wordt? Wat gebeurd er dan met de textarea?